app/Http/Controllers/GetArticleController.php

```diff
@@ -23,17 +24,14 @@ public function __construct(GetArticleHandler $handler)
         $this->handler = $handler;
     }
 
-    public function __invoke(string $id)
+    /**
+     * @throws ArticleNotFound
+     */
+    public function __invoke(string $id): JsonResponse
     {
         try {
             $command = new GetArticleCommand(ArticleID::fromUUID($id));
             $article = $this->handler->__invoke($command);
```

Comment:
Use the try catch only for the proper place where the exception may be thrown.
This will prevent a possible issue in the future.

```suggestion
 try {
            $articleID = ArticleID::fromUUID($id)
            ...
catch (){...}
$command = new GetArticleCommand($articleID);
 $article = $this->handler->__invoke($command);
```

Reply:
I will remove that try catch too, because that's is an InvalidInput, or ValidationError, following the same logic of EntityNotFound